srcpkgs/python3-pytest-forked/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-pytest-forked'
+pkgname=python3-pytest-forked
+version=1.4.0
+revision=1
+wrksrc="${pkgname#python3-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-setuptools_scm"
```

Comment:
Just `python3-setuptools_scm` should be fine here, no need for also including `python3-setuptools`

Reply:
I'd rather spell it out, in case the need for `_scm` is removed later.

Reply:
fair enough, although a quick grep of `srcpkgs` looks to me like we're not doing that in most cases